py17track/package.py

```diff
@@ -239,9 +239,11 @@
 
 PACKAGE_STATUS_MAP: Dict[int, str] = {
     0: "Not Found",
+    5: "Unknown",
     10: "In Transit",
     20: "Expired",
     30: "Ready to be Picked Up",
+    32: "Unknown",
```

Comment:
We can cover these and other potentially unknown values by, instead, changing line 284:

```python
object.__setattr__(self, "status", PACKAGE_STATUS_MAP.get(self.status, "Unknown"))
```

Reply:
Just gave it a try and it doesn't work if I remove my fix and the error happens before even creating the Package class

Reply:
applied that suggestion to line 111 in profile.py and it works